### PR TITLE
Adding CryptoPhoto as a 2FA provider

### PR DESCRIPTION
**Pull request by gitmanju**

### CODE REVIEW

File: _data/providers.yml [hunk 1]
```diff
@@ -42,7 +42,14 @@ providers:
     url: https://getclef.com
     img: clef.png
     software: Yes
-
+    
+  - name: CryptoPhoto
+    url: https://www.cryptophoto.com
+    img: crypto.png
```
Comment (Carlgo11) [inline _data/providers.yml:48]: It would be good if you named the logo cryptophoto.png instead.

**Comment by Carlgo11** [comment]
You don't need to submit a new PR every time you change something. Just push new commits to the branch you're working in and they'll be uploaded to the PR.

File: _data/providers.yml (hunk 1 above)
Comment (gitmanju) [inline _data/providers.yml:48]: ok, will do now. When will it show in the web ?

Comment (Carlgo11) [inline _data/providers.yml:48]: When you've fixed this I'll label this PR as `Looks good`. Then another collaborator needs to verify it before merging.
The owner later checks that everything looks nice before he updates the site. It usually takes 1-5 days.
Could take longer depending on how much work we have outside of this project.  
We're all volunteers and do this in our spear time so please have patience.

Comment (gitmanju) [inline _data/providers.yml:48]: Thanks carlgo11; I changed the file. As suggested I only did push and commit (bear with me, this is my use of github).

Comment (Carlgo11) [inline _data/providers.yml:48]: mm well you could just rename crypto.png to cryptophoto.png. You just uploaded a 646x224px JPEG. That's not a 32x32px png ;)
Please remove the new one and rename the old picture.

And no worries, we all have to be noobs sometimes. As long as you don't scream at me I'm good :smile:

Comment (gitmanju) [inline _data/providers.yml:48]: oops :-) I had renamed it, not sure what happened, let me upload it once again !! Thanks :-)

**Comment by mxxcon** [comment]
@gitmanju Site icons should have the same names as sites themselves so that it's easy to identify what belongs to what. These icons should be PNG image 32x32.
What you need to do is: 
- delete existing huge logo file
- rename `crypto.png` into `cryptophoto.png` 
- update its reference in your yaml block

**Pull request closed without merging**